pygfx/renderers/wgpu/shaders/meshshader.py

```diff
@@ -179,6 +188,33 @@ def get_bindings(self, wobject, shared):
         sampler = GfxSampler(material.map_interpolation, "repeat")
         texturing = "texture/auto"
 
+        # todo, check uv is present and has the right shape. (introduce uv channel for texture)
+        # specular map configs, for basic and phong materials
+        if getattr(material, "specular_map", None) and not isinstance(
+            material, MeshStandardMaterial
+        ):
```

Comment:
Is this todo still relevant?

Also maybe add a comment on why this `if` is not entered for `MeshStandardMaterial`.

Reply:
> Also maybe add a comment on why this `if` is not entered for `MeshStandardMaterial`.

Ah this is redundant code (overlooked during cherrypicking from #587). 😅 


We can introduce the `specular_map` property for `MeshBasicMaterial` and `MeshPhongMaterial`. The lighting model of `MeshPhongMaterial` requires the `specular_strength` parameter. Additionally, the shading logic for `env_map` in `MeshBasicMaterial` and `MeshPhongMaterial` also requires the `specular_strength` parameter. Currently, `specular_strength` is hardcoded to 1.0 but can be read from `specular_map`.

However, the lighting model of `MeshStandardMaterial` does not require `specular_strength`, and its `env_map` uses IBL (Image-Based Lighting) technology, thus does not need the `specular_strength` parameter.

 I will remove this code in this PR, and I plan to add support for `Specular_map` to `MeshBasicMaterial` and `MeshPhongMaterial` in another PR.